schema.sql

```diff
@@ -4,5 +4,8 @@ CREATE TABLE animals(
     date_of_birth DATE NOT NULL,
     escape_attempts INT,
     neutered BOOLEAN NOT NULL,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,    
 );
```

Comment:
- [ ] I would advise you to remove the **_comma (,)_** after this line of code `weight_kg DECIMAL`. It is a bug that produces the syntax error message and consequently, the `animals` table can not be created. 😎 💻 

![image](https://user-images.githubusercontent.com/77106962/228270788-7cbec269-617a-47df-a6f6-ac6023a38d45.png)